generate_grpc.sh

```diff
@@ -18,10 +18,23 @@ if [ ! -e ${DIR} ]; then
     wget https://raw.githubusercontent.com/longhorn/longhorn-engine/master/proto/ptypes/syncagent.proto -P $DIR
 fi
 
-# instance manager
+# common
+python3 -m grpc_tools.protoc -I pkg/imrpc -I proto/vendor/protobuf/src/ --python_out=integration/rpc/common --grpc_python_out=integration/rpc/common pkg/imrpc/imcommon.proto
+protoc -I pkg/imrpc/ -I proto/vendor/protobuf/src/ pkg/imrpc/imcommon.proto --go_out=plugins=grpc:pkg/imrpc
+
+# instance manager (process manager)
 python3 -m grpc_tools.protoc -I pkg/imrpc -I proto/vendor/protobuf/src/ --python_out=integration/rpc/instance_manager --grpc_python_out=integration/rpc/instance_manager pkg/imrpc/imrpc.proto
 protoc -I pkg/imrpc/ -I proto/vendor/protobuf/src/ pkg/imrpc/imrpc.proto --go_out=plugins=grpc:pkg/imrpc
 
 # proxy
 python3 -m grpc_tools.protoc -I pkg/imrpc -I proto/vendor/ -I proto/vendor/protobuf/src/ --python_out=integration/rpc/proxy --grpc_python_out=integration/rpc/proxy pkg/imrpc/proxy.proto
 protoc -I pkg/imrpc/ -I proto/vendor/ -I proto/vendor/protobuf/src/ pkg/imrpc/proxy.proto --go_out=plugins=grpc:pkg/imrpc/
+
+# disk
+python3 -m grpc_tools.protoc -I pkg/imrpc -I proto/vendor/ -I proto/vendor/protobuf/src/ --python_out=integration/rpc/disk --grpc_python_out=integration/rpc/disk pkg/imrpc/disk.proto
+protoc -I pkg/imrpc/ -I proto/vendor/ -I proto/vendor/protobuf/src/ pkg/imrpc/disk.proto --go_out=plugins=grpc:pkg/imrpc/
```

Comment:
Let's make sure proto registration is unique. ref: https://github.com/longhorn/backing-image-manager/commit/38ff27cc

Reply:
Addressed. Thank you.